x-pack/test/saved_object_api_integration/common/suites/bulk_delete.ts

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SuperTest } from 'supertest';
+import type { Client } from '@elastic/elasticsearch';
+import expect from '@kbn/expect';
+import type { SearchTotalHits } from '@elastic/elasticsearch/lib/api/types';
+import { SAVED_OBJECT_TEST_CASES as CASES } from '../lib/saved_object_test_cases';
+import { SPACES } from '../lib/spaces';
+import { expectResponses, getUrlPrefix, getTestTitle } from '../lib/saved_object_test_utils';
+import { ExpectResponseBody, TestCase, TestDefinition, TestSuite, TestUser } from '../lib/types';
+
+export interface BulkDeleteTestDefinition extends TestDefinition {
```

Comment:
The tests for the bulk operation were based on those for single `delete` and modified accordingly.